.pipelines/templates/PackageBuild.yml

```diff
@@ -194,6 +198,8 @@ steps:
         $delta_fetch_arg \
         $quick_rebuild_packages_arg \
         $run_check_arg \
+        TEST_RERUN_LIST="${{ parameters.testRerunList }}" \
+        $rerun_toolchain_checks_arg \
```

Comment:
I think we didn't define `rerun_toolchain_checks_arg` anywhere.

Reply:
Eugh, didn't save those two changes before committing... fixed now.